tests/autoyast/installation.pm

```diff
@@ -173,15 +175,15 @@ sub run {
     $checktime = 30;
     $looptime  = 0;
     $timeout   = 0;
-    while (!$timeout) {
+    while ( !$timeout ) {
 
         mouse_hide(1);
-        $ret = check_screen(["reboot-after-installation", "autoyast-error"], $checktime);
+        $ret = check_screen( [ "reboot-after-installation", "autoyast-error" ], $checktime );
 
         #repeat until timeout or login screen
-        if (defined $ret) {
-            if (match_has_tag('autoyast-error')) {
-                record_soft_failure 'AUTOYAST_EXPECT_ERRORS ' . get_var('AUTOYAST_EXPECT_ERRORS_REASON');
+        if ( defined $ret ) {
+            if ( match_has_tag('autoyast-error') ) {
+                diag( 'Expected error. Iteration = ' . $i );
```

Comment:
1. I wrote you to use record_info instead of diag and also leave record_soft_failure, how is this making me 'trying to move out soft failure from git' ?
2. Anybody has to be aware of any existing VARIABLE, anybody who is starting to use openQA should first copy/paste existing tests to understand tests and test variables in openQA
3. AUTOYAST_EXPECT_ERRORS_REASON is doing nothing else than informing anybody who is looking on test why it was soft_failing so get the reason of the workaround. It can hold info about one or then bugs, doesn't matter. AUTOYAST_EXPECT_ERRORS_REASON=bsc#1 bsc#200 bsc#whatever